xskillscore/core/deterministic.py

```diff
@@ -173,6 +173,11 @@ def pearson_r(a, b, dim, weights=None, skipna=False):
         Weights matching dimensions of ``dim`` to apply during the function.
     skipna : bool
         If True, skip NaNs when computing function.
+    keep_attrs : bool
+        If True, the attributes (attrs) will be copied
+        from the original object to the new one.
```

Comment:
Maybe clarify in the description that it's copying them from `a`. From my interpretation of `xr.apply_ufunc` docs it copies the attrs from the first argument.